persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageMapper.kt

```diff
@@ -176,16 +176,24 @@ object MessageMapper {
                 adminName = senderName
             )
 
-            MessageEntity.ContentType.UNKNOWN -> MessagePreviewEntityContent.Unknown
-            MessageEntity.ContentType.FAILED_DECRYPTION -> MessagePreviewEntityContent.Unknown
             MessageEntity.ContentType.REMOVED_FROM_TEAM -> MessagePreviewEntityContent.TeamMemberRemoved(userName = senderName)
-            MessageEntity.ContentType.CRYPTO_SESSION_RESET -> MessagePreviewEntityContent.CryptoSessionReset
+
+            MessageEntity.ContentType.COMPOSITE -> MessagePreviewEntityContent.Text(
+                senderName = senderName,
+                // composite messages can have not text since they are just a collection of Text and Buttons
+                // so it the collection have no text we just set it is null therefore no need to require it
```

Comment:
But what if there's no `text`, what should be shown in the preview then? Wouldn't be better to provide some other preview type for composite and in case it has no text, then we could show something like "someone sent a poll message"

Reply:
Composite messages are not only poll messages, I was considering using the same copy as ephemeral messages `Someone sent a message`

Reply:
There's no guarantee it is a `poll` either. Composite Messages are _not_ exclusive for polls. They could be for controlling music playlists as far as we know.
Maybe "interactive message" would be a better name for messages with no text but with buttons.

Reply:
ok then i will create a new type for composite message preview and revert the change to nullable text